lms/templates/learner_dashboard/certificate.underscore

```diff
@@ -0,0 +1,7 @@
+<div class="certificate-container">
+    <p class="title"><%- gettext('XSeries Program Certificates') %>:</p>
+    <img src="/static/images/xseries-certificate-visual.png" alt="XSeries Program Certificates" aria-hidden="true">
```

Comment:
@ahsan-ul-haq as per http://www.w3schools.com/tags/att_img_alt.asp because the image is only used for decoration we wouldn't want anything to render if the image failed to load. The text around the image provides enough context for the user as it is.